config/routes.rb

```diff
@@ -1,74 +1,21 @@
 Rails.application.routes.draw do
-  root  'urls#new'
+  root  " urls#index"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.